cli/main.rs

```diff
@@ -445,15 +445,33 @@ async fn bundle_command(
     let output_bytes = output_string.as_bytes();
     let output_len = output_bytes.len();
     deno_fs::write_file(out_file_, output_bytes, 0o666)?;
-    // TODO(bartlomieju): add "humanFileSize" method
-    info!("{} bytes emitted.", output_len);
+    info!("{} emitted.", human_size(output_len as f64));
   } else {
     println!("{}", output_string);
   }
-
   Ok(())
 }
 
+fn human_size(bytse: f64) -> String {
```

Comment:
Can you add a unit test next to this that checks the output of human size?

Reply:
sure, added 👍